src/controllers/auth.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from 'express'
+import jwt from 'jsonwebtoken'
+import bcrypt from 'bcrypt'
+import User from '../models/index'
+import Joi from 'joi'
+import config from 'dotenv'
+export const loginUser = async (req: Request, res: Response) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).json({ error: error.details[0].message })
+  const { email, password } = req.body
+
+  try {
+    const user = await User.findOne({ where: { email } })
+
+    if (!user) {
+      return res.status(404).json({ message: "User doesn't exist" })
+    }
+
+    const passwordMatch = await bcrypt.compare(password, user.password)
+
+    if (!passwordMatch) {
+      return res.status(401).json({ message: 'Invalid email or password' })
+    }
+    const token = jwt.sign({ email: user.email }, process.env.JWTTOKEN || '')
```

Comment:
this doesn't belong here. It's a utility. Create a utils folder then jwt.utils file then add all of the jwt logic there. generating token, verifying, etc...